udata/rdf.py

```diff
@@ -301,22 +302,64 @@ def contact_point_from_rdf(rdf, dataset):
             or rdf_value(contact_point, VCARD.email)
             or rdf_value(contact_point, DCAT.email)
         )
-        if not email:
-            return
         email = email.replace("mailto:", "").strip()
+        contact_form = rdf_value(contact_point, VCARD.hasUrl)
+        if not email and not contact_form:
+            return
```

Comment:
We're now requiring either an email OR a contact_form to have a proper contact point.